packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -114,10 +112,58 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
               data-ui-id="meeting-composite-people-button"
               disabled={props.disableButtonsForLobbyPage}
               label={meetingStrings.peopleButtonLabel}
+              styles={commonButtonStyles}
+            />
+          )}
+          {isEnabled(meetingCallControlOptions?.chatButton) !== false && (
+            <ChatButton
+              checked={props.chatButtonChecked}
+              showLabel={true}
+              onClick={props.onChatButtonClicked}
+              data-ui-id="meeting-composite-chat-button"
+              disabled={props.disableButtonsForLobbyPage}
+              label={meetingStrings.chatButtonLabel}
+              styles={commonButtonStyles}
             />
           )}
-        </Stack.Item>
+        </Stack>
       )}
     </Stack>
   );
 };
+
+const buttonContainerStyle: IStyle = {
+  padding: '0.75rem',
+  columnGap: '0.5rem'
+};
+
+const controlBarStyles: BaseCustomStyles = {
+  root: buttonContainerStyle
+};
+
+const commonButtonStyles: ControlBarButtonStyles = {
+  root: {
+    border: 'solid 1px #E1DFDD',
+    borderRadius: '0.25rem',
+    minHeight: '2.5rem'
+  },
+  flexContainer: {
+    flexFlow: 'row nowrap'
+  },
+  textContainer: {
+    // Override the default so that label doesn't introduce a new block.
+    display: 'inline'
+  },
+  label: {
+    // Override styling from ControlBarButton so that label doesn't introduce a new block.
+    display: 'inline',
+    fontSize: '0.875rem'
```

Comment:
This breaks when end user changes the default font size using theme.
I recommend using
`theme.fonts.small.size...` (something like that)

Reply:
Done